.kiro/specs/pr-metrics-action/design.md

```
@@ -0,0 +1,1556 @@
+
+**目的**: PR Metrics Actionは、プルリクエストのサイズと複雑度を自動的に評価し、レビューの効率化を実現する。
+
+**ユーザー**: 開発チームはこれを利用して、大きすぎるPRの早期発見と適切な分割判断を行う。
+
+**影響**: 現在手動で行われているPRサイズの確認を自動化し、一貫性のある品質基準を確立する。
+
+### ゴール
+
+- PRのファイルメトリクス（サイズ、行数、ファイル数）を自動計測
+- 制限超過時の自動ラベル付与とコメント投稿
+- CI/CDパイプラインとのシームレスな統合
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)